src/DiscreteFilter.cpp

```diff
@@ -0,0 +1,314 @@
+/*
+ * DiscreteFilter.cpp
+ *
+ *  Created on: 19/09/2014
+ *      Author: rafaelsaback
+ */
+
+#include "DiscreteFilter.hpp"
+
+namespace base {
```

Comment:
The namespace of this library is currently 'base' while the library is 'numeric'.
Though this has been done for the other parts of the library, I would vote for start using the 'numeric' namespace here (for new classes) -- that is actually a question to the other rock-devs / maintainer. Otherwise, this functionality 'should/would' go into the actual base-lib!?